core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2225,13 +2225,15 @@ object Log extends Logging {
    * @param dir The directory in which the log will reside
    * @param topicPartition The topic partition
    * @param logDirFailureChannel The LogDirFailureChannel to asynchronously handle log dir failure
-   * @param recordVersion The record version
+   * @param recordVersion The record
```

Comment:
Is it intended to change the `The record version` to `The record`?

Reply:
No. It was changed accidentally. I have fixed it now in 35e389b5c05fddf5682f344ddf58222603133a19.